### PR TITLE
crystals drop shards and ore crab tweaks.

### PR DESCRIPTION
**Pull request by brainfood1183**
Added - crystals now drop shards.
Added - colored crystal light tubes can be crafted from crystal shards.
Added - silver ore crab added.
Tweaked - some ore crabs had their damage lowered.
Changed - uranium ore crab will now attack back if provoked.
Changed - crystals can now be unanchored.

untitled-338

- [x] Redo the shard sprites to look better.
- [x] Currently light tube fixtures have a default tube style sprite when filled which seems out of place for a shard.
- [x] Possibly integrate the shards into some crafting recipe.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Crystals now drop crystal shards, you can use them to craft colored light tubes.

### CODE REVIEW

**Comment by GoodWheatley** [comment]
This PR isn’t actually touching the melee damage values for ore crabs, is that unintentional?

**Comment by EmoGarbage404** [comment]
crystals can 100% be a single sprite with recoloring in yaml.

**Comment by brainfood1183** [comment]
> crystals can 100% be a single sprite with recoloring in yaml.

yea, wil do that

**Comment by Emisse** [comment]
i accidentally conflicted you with #20997 its a small change

**Review by metalgearsloth — DISMISSED** [review]
Didn't finish my review yet but this was the start.

File: Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml [hunk 1]
```diff
@@ -76,27 +76,40 @@
   - type: ZombieImmune
 
 - type: entity
+  save: false
```
[the remark this answers is not in the document]
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml]: removed the save: false

File: Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml [hunk 2]
```diff
@@ -76,27 +76,40 @@
...
+  name: ore crab
   parent: [ MobElementalBase, MobCombat ]
-  id: MobQuartzCrab
-  name: quartz crab
-  description: An ore crab made from quartz.
   components:
   - type: Sprite
     sprite: Mobs/Elemental/orecrab.rsi
-    state: quartzcrab
+    state: uraniumcrab
```
[the remark this answers is not in the document]
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml]: snake_cased the sprite names

File: Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml [hunk 3]
```diff
@@ -196,15 +199,48 @@
...
+  - type: Sprite
+    sprite: Mobs/Elemental/orecrab.rsi
+    state: silvercrab
+  - type: MeleeWeapon
+    hidden: true
+    soundHit:
+      path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
+    damage:
+      types:
+        Blunt: 5
```
[the remark this answers is not in the document]
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml]: removed parented variables

File: Resources/Prototypes/Entities/Objects/Power/lights.yml [hunk 4]
```diff
@@ -113,6 +144,37 @@
...
+      - !type:PlaySoundBehavior
+        sound:
+          collection: GlassBreak
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          ShardGlass:
+            min: 1
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Power/lights.yml:177]: why is this not just on the parent

**Review by mirrorcult — DISMISSED** [review]

File: Resources/Prototypes/Entities/Objects/Power/lights.yml (hunk 4 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Objects/Power/lights.yml]: Sorry realized i had made pr but hadnt actually pushed it, now pushed.

**Pull request merged**